solutions/cf-782b.mdx

```diff
@@ -101,4 +101,53 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+num_friends = int(input())
+friend_coords = list(map(int, input().split()))
+friend_veloci = list(map(int, input().split()))
+
+
+def all_friends_converge(seconds: int) -> bool:
+	"""
+	Checks whether all friends can converge on one point in the specified time interval.
+
+	Args:
+		seconds (int): Amount of seconds given for friends to converge.
+
+	Returns:
+		bool: If the friends can converge to a single point.
+	"""
```

Comment:
```suggestion
	"""
	Checks whether all friends can converge on one point in the specified time interval.
	:param seconds: Amount of seconds given for friends to converge.
	:returns: If the friends can converge to a single point.
	"""
```
doc comment syntax from [here](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats)

Reply:
oh wait this seems like the google syntax
uh most of the docstrings here follow the reST syntax